files/check_openvpn

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Checks if an OpenVPN is avalilable and working.
+
+# Parameters:
+# -c - OpenVPN configuraton that should be used
+# -h - Host name that should be reached through the openvpn
+
+. "$(dirname "${0}")/common.sh"
```

Comment:
might want to use "${BASH_SOURCE[0]} instead of $0", as it produces the same behavior regardless of whether the script is invoked as <name> or source <name>